### PR TITLE
NEPT-1746: Add missing component 'user_picture' and other related user components.

### PR DESCRIPTION
**Pull request by drupol**

### CODE REVIEW

File: atomium/templates/user_picture/user_picture.component.inc [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+/**
+ * Implements hook_preprocess_hook().
+ */
+function atomium_preprocess_user_picture(&$variables, $hook) {
+  if (variable_get('user_pictures', 0)) {
+    $account = $variables['account'];
+
+    if (!empty($account->picture)) {
+      // @TODO: Ideally this function would only be passed file objects, but
```
Comment (steeff) [inline atomium/templates/user_picture/user_picture.component.inc:31]: is this TODO still useful? if yes its purpose is not pretty clear...

Comment (drupol) [reply atomium/templates/user_picture/user_picture.component.inc:31]: It's the comment from the template_preprocess_comment() function. 

I don't think it hurts to leave it there as we've copy paste the code. 

Now, I will ask the team what they think and give you feedback asap. 

Thanks.

File: atomium/templates/node/node.tpl.php [hunk 2]
```diff
@@ -35,19 +35,13 @@
 
   <?php if ($display_submitted): ?>
       <footer>
-        <?php print $user_picture; ?>
```
Comment (imanoleguskiza) [inline atomium/templates/node/node.tpl.php]: The rest of the PR is great but: why are we removing this lines (this specific one and the ones at the bottom) from the node.tpl? I think I'm missing some extra information to understand this change.

**Review by imanoleguskiza — CHANGES_REQUESTED** [review]
I have just a small remark, once clarified the rest of the PR is great.

File: atomium/templates/node/node.tpl.php (hunk 2 above)
Comment (drupol) [reply atomium/templates/node/node.tpl.php]: Hi,

Most use cases doesn't need to print the user picture inside the node template, this is why it has been removed.

Comment (imanoleguskiza) [reply atomium/templates/node/node.tpl.php]: I still find it weird that we remove functionality without it being asked :( Anyways I'm not going to block the issue but I'd like to have a second opinion on this, so I'll keep the review open but move the ticket forward

Comment (drupol) [reply atomium/templates/node/node.tpl.php]: As you wish bruh'

Comment (voidtek) [reply atomium/templates/node/node.tpl.php]: I agree with @imanoleguskiza - is weird to rmove functionnality, furthermore in ec_resp we have the picture. So for me, we need to keep the picture.

Comment (drupol) [reply atomium/templates/node/node.tpl.php]: Ok, cancelling this change now.

**Review by imanoleguskiza — APPROVED** [review]

**Review by voidtek — APPROVED** [review]
The PR is fine for me.

**Pull request merged**